contrib/cirrus/runner.sh

```diff
@@ -233,6 +233,17 @@ function _run_consistency() {
     SUGGESTION="run 'make generate-bindings' and commit all changes" ./hack/tree_status.sh
     make completions
     SUGGESTION="run 'make completions' and commit all changes" ./hack/tree_status.sh
+    if  [[ -z "$CIRRUS_TAG" ]] && \
+        req_env_vars CIRRUS_CHANGE_IN_REPO CIRRUS_PR DEST_BRANCH &> /dev/null
+    then
```

Comment:
General comment: I find the logs from this script impossible to read. How about:
```bash
echo
echo "Checking for added ioutils"
```
...with a corresponding `else echo "Skipping check for ioutils"`? (the `else` is important because it differentiates "test passed" from "test never ran at all").

Reply:
good idea.